library/src/main/scala/giter8/GitRepository.scala

```diff
@@ -47,7 +47,7 @@ object GitRepository {
     val NativeUrl: Regex = "^(git[@|://].*)$".r
     val HttpsUrl: Regex  = "^(https://.*)$".r
     val HttpUrl: Regex   = "^(http://.*)$".r
-    val SshUrl: Regex    = "^(ssh://.*)$".r
+    val SshUrl: Regex    = "^ssh://(.*)$".r
```

Comment:
I thought a bit more about this, and I still feel icky about modifying the regex based on JGit's implementation because that's now how `git` works AFAIK.
I think a better place to put this in is https://github.com/foundweekends/giter8/blob/547db187ceadb4cefc504733ce47225f7b7b47aa/library/src/main/scala/giter8/GitInteractor.scala#L46-L70 as `def changeUrl(String): String` with a code comment explaining that it's a workaround.